### DISCUSSION TITLE
VTI config Mark not  undstood, hows does 42 =  0x01000201

### DISCUSSION THREAD

**Discussion by simondbingham** [strongswan/strongswan#758] [Q&A]
I do not understand the documentation as below , how does fo the value 42 in the example relate to the hex value in the example to my calcuation this should be 0x0000002A, Thanks 

### From Documentation ###
https://wiki.strongswan.org/projects/strongswan/wiki/RouteBasedVPN
_The most important configuration option is the mark (mark_in|out in swanctl.conf, mark in ipsec.conf). After applying the optional mask (default is 0xffffffff) to the mark that's set on the VTI device and it applied to the routed packets, the value has to match the configured mark. So referring to the example above, to match the mark on vti0 configure mark_in = mark_out = 42 and to match the mark on ipsec0 set the value to 0x01000201 (but something like 0x00000200/0x00000f00 would also work)._

**Comment by tobiasbrunner** [comment]
It doesn't, and why would it, it's a different interface that uses a different key/mark value (vti0 vs. ipsec0).
